### PR TITLE
Remove Blockage and Wedge blockage fix #3962

### LINKED ISSUE #3962: BUG: The "Remove Blockage" verb  doesn't work properly
BUG: The "Remove Blockage" verb  doesn't work properly, it only pops up a window with mob names.

https://discordapp.com/channels/255035529085583360/375588651599396866/632433104061071381

### PR DESCRIPTION
**Pull request by AlexMorgan3817**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>
<ul>
<li>Remove Blockage and Wedge blockage fixed by deleting arguments of user in this verbs

<li>if you ALT + click on blockaged airlock, you will remove blockage, but if it isn't blocked and in ur active hand crowbar, u'll block it. 
</ul>
<hr>
</details>

## Changelog
:cl:
add: Feature, if you ATL + click on blockaged airlock, you will remove blockage, but if it isn't blocked and in ur active hand crowbar, u'll block it. 
fix: Remove Blockage and Wedge blockage fixed
/:cl:

### CODE REVIEW

**Comment by AlexMorgan3817** [comment]
R4R

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -587,46 +587,46 @@ There are 9 wires.
...
 	set name = "Wedge item"
 	set category = "Object"
 	set src in view(1)
 
-	if(!user)
-		user = usr
-
-	var/obj/item/weapon/tool/T = user.get_active_hand()
+	if(!usr)
+		log_debug("Verb 'Wedge item' bugged and called without usr.")
```
Comment (comma) [inline code/game/machinery/doors/airlock.dm]: pretty sure usr is ALWAYS present, something is calling the proc after all.
The real check (if you REALLY want to use unreliable ass 'usr') would be checking if usr is mob.

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -587,46 +587,46 @@ There are 9 wires.
 	verbs -= /obj/machinery/door/airlock/proc/try_wedge_item
 	verbs += /obj/machinery/door/airlock/proc/take_out_wedged_item
 
-/obj/machinery/door/airlock/proc/try_wedge_item(mob/living/user)
+/obj/machinery/door/airlock/proc/try_wedge_item()
```
Comment (comma) [inline code/game/machinery/doors/airlock.dm]: Why is removing passing actual user fixing it? That makes no sense.

Comment (AlexMorgan3817) [reply code/game/machinery/doors/airlock.dm]: This removed, cuz this isn't proc, it's proc-verb, and if u put in () of verb any arguments, u will see input as mob and etc.

File: code/game/machinery/doors/airlock.dm (hunk 1 above)
Comment (AlexMorgan3817) [reply code/game/machinery/doors/airlock.dm]: U can't present argument of mob/user in verb

**Comment by AlexMorgan3817** [comment]
TODO: Replace if(!usr) by if(!istype(usr, /mob/living))

**Comment by AlexMorgan3817** [comment]
r4r

File: code/game/machinery/doors/airlock.dm [hunk 3]
```diff
@@ -587,46 +587,46 @@ There are 9 wires.
...
 	set name = "Wedge item"
 	set category = "Object"
 	set src in view(1)
 
-	if(!user)
-		user = usr
-
-	var/obj/item/weapon/tool/T = user.get_active_hand()
+	if(!istype(usr, /mob/living))
```
Comment (mattroks101) [inline code/game/machinery/doors/airlock.dm]: isliving(usr)

File: code/game/machinery/doors/airlock.dm [hunk 4]
```diff
@@ -587,46 +587,46 @@ There are 9 wires.
...
-/obj/machinery/door/airlock/proc/take_out_wedged_item(mob/living/user)
+/obj/machinery/door/airlock/proc/take_out_wedged_item()
 	set name = "Remove Blockage"
 	set category = "Object"
 	set src in view(1)
 
-	if(!user)
-		user = usr
-
+	if(!istype(usr, /mob/living))
```
Comment (mattroks101) [inline code/game/machinery/doors/airlock.dm]: `isliving(usr)`

**Comment by AlexMorgan3817** [comment]
Ready for merge

**Pull request merged**